recipe/meta.yaml

```diff
@@ -54,34 +54,37 @@ outputs:
     requirements:
       host:
         - sysroot_{{ ctng_target_platform }} {{ conda_glibc_ver }}
-        - {{ pin_subpackage("libgomp", exact=True) }}
-        - {{ pin_subpackage("libstdcxx-ng", exact=True) }}
-        - {{ pin_subpackage("libgcc-ng", exact=True) }}
-        - {{ pin_subpackage("libgfortran-ng") }}
```

Comment:
Why did you remove the exact pins in host here?

Reply:
In particular, if the build picks up older versions of these packages with different layouts, I am not sure if the final compiler would be right.

Reply:
Or older builds at the same version that change the layout. 

Also, I guess we are rely on build number priority to link all of the packages that have the same relative layouts? 

Seems fine but maybe exact pins would be safer?